test/calculate-svg-dimensions.test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const assert = require('assert').strict;
+const { readFile } = require('fs').promises;
+const path = require('path');
+const BrowserManager = require('../lib/browser-mananger.js');
+const calculateSvgDimensions = require('../lib/svg-sprite/calculate-svg-dimensions.js');
+
+describe('calculateSvgDimensions', () => {
+    let browserManager;
+
+    before(async() => {
+        browserManager = new BrowserManager();
+    });
+
+    after(() => browserManager.closeBrowser());
+
+    it('should return the svg dimensions', async() => {
+        const svgFilePath = path.join(__dirname, 'fixture/svg/weather-clear-dimension-calculation.svg');
+        const svg = await readFile(svgFilePath, 'utf-8');
+        const dimensions = await calculateSvgDimensions(svg, await browserManager.getBrowser());
+        const expected = { width: 43, height: 43 };
+
+        assert.deepEqual(dimensions, expected);
+    });
```

Comment:
It's all good but where is one moment. Because we treat `calculateSvgDimensions` function like a black box in tests we need to be sure if this function is not returns a hardcoded result of `{ width: 43, height: 43 }`
So, can you please add another fixture with another expected dimensions?
And it'll be great to see a test which checks the consistency of results. So in one test case we expecting that this function returns same result.